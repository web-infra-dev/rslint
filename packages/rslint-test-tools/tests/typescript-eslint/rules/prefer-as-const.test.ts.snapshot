exports[`prefer-as-const > invalid 1`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` instead of a literal type assertion.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 27
        },
        "end": {
          "line": 1,
          "column": 32
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 10`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` instead of a literal type assertion.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 16
        },
        "end": {
          "line": 1,
          "column": 17
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 11`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` assertion instead of a literal type annotation.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 3,
          "column": 8
        },
        "end": {
          "line": 3,
          "column": 13
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 12`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` assertion instead of a literal type annotation.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 3,
          "column": 8
        },
        "end": {
          "line": 3,
          "column": 9
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 13`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` instead of a literal type assertion.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 3,
          "column": 10
        },
        "end": {
          "line": 3,
          "column": 15
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 14`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` instead of a literal type assertion.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 3,
          "column": 18
        },
        "end": {
          "line": 3,
          "column": 23
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 15`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` instead of a literal type assertion.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 3,
          "column": 14
        },
        "end": {
          "line": 3,
          "column": 15
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 2`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` instead of a literal type assertion.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 23
        },
        "end": {
          "line": 1,
          "column": 24
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 3`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` assertion instead of a literal type annotation.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 9
        },
        "end": {
          "line": 1,
          "column": 14
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 4`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` assertion instead of a literal type annotation.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 10
        },
        "end": {
          "line": 1,
          "column": 15
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 5`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` assertion instead of a literal type annotation.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 10
        },
        "end": {
          "line": 1,
          "column": 11
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 6`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` instead of a literal type assertion.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 27
        },
        "end": {
          "line": 1,
          "column": 32
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 7`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` instead of a literal type assertion.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 12
        },
        "end": {
          "line": 1,
          "column": 17
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 8`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` instead of a literal type assertion.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 12
        },
        "end": {
          "line": 1,
          "column": 13
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`prefer-as-const > invalid 9`] = `
{
  "diagnostics": [
    {
      "ruleName": "prefer-as-const",
      "message": "Expected a \`const\` instead of a literal type assertion.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 20
        },
        "end": {
          "line": 1,
          "column": 25
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;
