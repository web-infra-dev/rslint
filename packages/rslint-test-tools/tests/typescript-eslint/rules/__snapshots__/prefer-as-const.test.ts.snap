// Rstest Snapshot v1

exports[`prefer-as-const > invalid 1`] = `
{
  "diagnostics": [],
  "errorCount": 0,
  "fileCount": 1,
  "ruleCount": 0,
}
`;
